posts/how-to-search-code-with-sourcegraph-a-cheat-sheet.md

```diff
@@ -128,7 +128,7 @@ after:time-period'
 matcher='time-period'
 /> 
 
-Sometimes the time period can be literal, like `last week`, `last year`, `3 months ago`, `february 10 2021` or have actual dates in the format `dd/mm/yyyy`.
+Sometimes the time period can be relative, like `last week`, `last year`, `3 months ago` or absolute, in several formats including `{month} {day} {year}` (example: `february 10 2021`), `dd/mm/yyyy`, and ISO format `yyyy-mm-dd`.
```

Comment:
I thought not everyone may be familiar with ISO so added "format" for context, feel free to contextualize more

Reply:
Also "relative" is a good call